src/ImageSharp.Web.Providers.Azure/Providers/AzureBlobStorageImageProvider.cs

```diff
@@ -63,25 +59,6 @@ public class AzureBlobStorageImageProvider : IImageProvider
             this.storageOptions = storageOptions.Value;
             this.formatUtilities = formatUtilities;
             this.storageAccount = CloudStorageAccount.Parse(this.storageOptions.ConnectionString);
-
-            try
```

Comment:
I'm ok dropping this. We should probably throw and exception if no container exists for the given name though.